roscala/src/main/scala/coop/rchain/rosette/utils/Instances.scala

```diff
@@ -0,0 +1,29 @@
+package coop.rchain.rosette.utils
+
+import coop.rchain.rosette.{Ob, TblObject}
+import shapeless._
+import record._
+import syntax.singleton._
+
+import scala.collection.mutable
+
+object Instances {
```

Comment:
Can you briefly describe what you are trying to achieve here?

Reply:
Here is Instance that helps Shapeless to derive an instance for a lens with a selector for `slot`.